packages/typescript/src/mutator/ConditionalExpressionMutator.ts

```diff
@@ -1,12 +1,31 @@
 import * as ts from 'typescript';
 
+import { printNode } from '../helpers/tsHelpers';
+
 import NodeMutator, { NodeReplacement } from './NodeMutator';
 
-export default class ConditionalExpressionMutator extends NodeMutator<ts.BinaryExpression> {
+/**
+ * Type guard for seperating default clause from case clauses.
+ */
+function isDefaultClause(node: ts.CaseOrDefaultClause): node is ts.DefaultClause {
+  return node.kind === ts.SyntaxKind.DefaultClause;
+}
+
+export default class ConditionalExpressionMutator extends NodeMutator<
+  ts.BinaryExpression | ts.DoStatement | ts.ForStatement | ts.IfStatement | ts.CaseOrDefaultClause | ts.WhileStatement
+> {
   public name = 'ConditionalExpression';
 
   public guard(node: ts.Node): node is ts.BinaryExpression {
-    return node.kind === ts.SyntaxKind.BinaryExpression;
+    return (
+      node.kind === ts.SyntaxKind.BinaryExpression ||
+      node.kind === ts.SyntaxKind.CaseClause ||
```

Comment:
Is case clause a conditional expression mutator thing? Should be added in the docs.